projects/admin/src/app/guard/can-add-local-fields.guard.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { LocalStorageService } from '@rero/ng-core';
+import { User } from '@rero/shared';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { LocalFieldApiService } from '../api/local-field-api.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CanAddLocalFieldsGuard implements CanActivate {
+
```

Comment:
It would be great to have a comment to explain what is the purpose of this guard.

Reply:
done